### DISCUSSION TITLE
运行报错问题

### DISCUSSION THREAD

**Discussion by GitHub0Yulin** [note286/xduts#96] [Q&A] [answered]
求问运行突然遇到“expl3.sty”文件错误怎么解决？
报错信息：
LaTeX3 Error: Mismatched LaTeX support files detected. \ExplLoaderFileDate{expl3.sty}}
Undefined control sequence. \sys_if_output_pdf:TF
LaTeX3 Error: Control sequence \ifctexpdf already defined. { \cs_new_eq:NN \ifctexpdf
![2023-03-23_200246](https://user-images.githubusercontent.com/49628703/227198118-d900b56f-4271-4bde-8285-e8e057d5a562.png)

**Comment by note286** [comment, marked as answer]
参考 https://github.com/note286/xduts/discussions/72 。
